src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -29,92 +29,48 @@ alt: mobile device with app panel open
-origin trial. After the origin trial ended, they took stock of the feedback and
-iterated on the design.
-
 <div class="w-clearfix"></div>
 
-## Suggested use cases {: #use-cases }
-
-* Checking for the native version of an app and switching to it
-* Disabling notifications in the web app when the native app is installed
-* Measuring how often users visit your website instead of your installed app
-* Not prompting users to install the web app if the native app is installed
-
-## Current status {: #status }
-
-<div class="w-table-wrapper">
-
-| Step                                         | Status                       |
-| -------------------------------------------- | ---------------------------- |
-| 1. Create explainer                          | [Complete][explainer]        |
-| 2. Create initial draft of specification     | [Complete][spec]             |
-| 3. Gather feedback and iterate on design     | Complete                     |
-| 4. Origin trial                              | Complete                     |
-| 5. Launch                                    | Complete                     |
+### Supported app types you can check
 
-</div>
+Your website can check if your:
 
-`getInstalledRelatedApps()` is supported on the following platforms:
+* [Android app](#check-android) is installed
+* [Windows (UWP) app](#check-windows) is installed
+* Progressive Web App is installed, if it's running in the
+  [same scope](#check-pwa-in-scope) or in a
+  [different scope](#check-pwa-out-of-scope)
 
-* **Check if your Android app is installed**
-  * Android: Chrome 80 or later
-* **Check if you PWA is installed**
-  * Android: Chrome 84 or later
-  * Windows: Edge 85 or later
+<!--  Android App -->
 
-Additional support is under development.
+## Check if your Android app is installed {: #check-android }
 
-## Define the relationship to your other apps {: #define-relationship }
+Your website can check if your Android app is installed.
 
-To use `getInstalledRelatedApps()`, you must first create a relationship
-between between your apps and sites. This relationship prevents other apps
-from using the API to detect if your app is installed and prevents sites from
-collecting information about the apps you have installed on your device.
+* Android: Chrome 80 or later
 
-In your [web app manifest](/add-manifest/), add a `related_applications`
-property. The `related_applications` property is an array containing an object
-for each app that you want to detect. Each app object includes:
+### Tell your Android app about your website
 
-* `platform` The platform on which the app is hosted
-* `id` The unique identifier for your app on that platform
-* `url` The URL where your app is hosted
+First, you'll need to update your Android app to define the relationship
+between your website and Android application using the
+[Digital Asset Links system][dig-asset-links]. This ensures that only your
+website can check if your Android app is installed.
```

Comment:
It's very rare that I suggest adding italics or bold to an article I'm reviewing. The goal should always be to add emphasis through word choice and sentence structure. I'm wondering if *your website* should be italicized.

Reply:
Fair point, I italicized them at the beginning, and added an aside that calls it out more specifically.